chat-widget/samples/javascript-sample/SampleWidget.js

```diff
@@ -12,123 +10,140 @@ import { version as chatComponentVersion } from "@microsoft/omnichannel-chat-com
 import { version as chatSdkVersion } from "@microsoft/omnichannel-chat-sdk/package.json";
 import { version as chatWidgetVersion } from "../../package.json";
 import { getCustomizationJson } from "./getCustomizationJson";
-import { memoryDataStore } from "./Common/MemoryDataStore";
 import getMockChatSDKIfApplicable from "./getMockChatSDKIfApplicable";
+import { memoryDataStore } from "./Common/MemoryDataStore";
 
 let liveChatWidgetProps;
 
-const main = async () => {
-
+async function fetchAuthTokenViaNetlify() {
+    const url = "https://omnichannel-auth-server.netlify.app/.netlify/functions/api/login";
+    const lwiContexts = {
+        Name: { value: "Contoso", isDisplayable: "true" },
+        Email: { value: "foo@microsoft.com", isDisplayable: "true" },
+        Secret: { value: "***" }
+    };
+    const data = {
+        contactid: "edward",
+        expiry: 10 * 60,
+        lwicontexts: JSON.stringify(lwiContexts)
+    };
+    const payload = {
+        method: "POST",
+        body: JSON.stringify(data)
+    };
+    try {
+        const response = await fetch(url, payload);
+        if (response.ok) {
+            const authToken = await response.text();
+            return authToken;
+        }
+    } catch (err) {
+        console.log(err);
+    }
+    return "";
+}
+
+async function main() {
     const queryString = window.location.search;
     const urlParams = new URLSearchParams(queryString);
     const orgId = urlParams.get("data-org-id");
     const orgUrl = urlParams.get("data-org-url");
     const appId = urlParams.get("data-app-id");
-
     const script = document.getElementById("oc-lcw-script");
     const customizationJson = await getCustomizationJson();
     const omnichannelConfig = {
         orgId: orgId ?? script?.getAttribute("data-org-id"),
         orgUrl: orgUrl ?? script?.getAttribute("data-org-url"),
         widgetId: appId ?? script?.getAttribute("data-app-id")
     };
-
-    let chatSDK = new OmnichannelChatSDK(omnichannelConfig);
+    const authToken = await fetchAuthTokenViaNetlify();
+    const chatSDKConfig = {
+        getAuthToken: () => authToken || ""
+    };
+    let chatSDK = new OmnichannelChatSDK(omnichannelConfig, chatSDKConfig);
```

Comment:
Using auth token to initialize the OmnichannelChatSDK wont necessarily make it perisistent chat right, however if it is persistent chat, the auth token callback fetchAuthTokenViaNetlify can be used. Is that correct?

Want to make sure the sample still works after this change for newly created, non-presistent chat widgets.